scripts/ci/3-setup-databases.sh

```diff
@@ -20,4 +20,9 @@ set -exuo pipefail
 
 MYSQL_HOST=mysql
 
+# make sure mysql is done setting up
+while ! mysqladmin ping -h ${MYSQL_HOST} -u root --silent; do
+    sleep 1
+done
```

Comment:
shouldn't the while loop need to exist after some time instead of looping forever?

Reply:
Probably a good idea.

Though also: why do we set up mysql when we are running against sqlite or postgres?

Reply:
> shouldn't the while loop need to exist after some time instead of looping forever?

done

> Though also: why do we set up mysql when we are running against sqlite or postgres?

done

Reply:
One reason might be to test the MySql operator etc, but we could probably just set that to skip those tests in that test matrix.

If it's more work to do (i.e. if you haven't done it yet, or the tests don't pass) then you could make that a separate PR

Reply:
should be good to go now